src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -244,10 +245,14 @@ const BrokerFormContent: FC<Props> = ({
               {status => (
                 <Input
                   type={InputType.Number}
-                  placeholder="1883"
+                  placeholder={`${
+                    !formContent || isNaN(formContent.brokerPort)
+                      ? DEFAULT_PORT
+                      : formContent.brokerPort
+                  }`}
```

Comment:
I think we can probably just remove the placeholder altogether now that the value is defaulted. I think it can be confusing for the user to tell if the placeholder is the default value or not. So it's probably better to just default to the default port value and if the user erases it, let the box be empty.

Reply:
Hmm I'm torn on this one. I feel like this box should become inconsistent from the other ones because it's the only one that has a default value. But I'm fine with leaving the placeholder for now and getting feedback from others before changing it

Reply:
@cannonpalms Tie-Breaker please 😆

Reply:
Not a fan of the placeholder, personally, as it leads the user to believe the form has already been filled in. My vote is to nuke it.

Reply:
## BEFORE
![BEFORE](https://user-images.githubusercontent.com/1637395/186769284-2c509fb6-0cf2-43fa-b5ef-4376f45eef56.gif)

## AFTER
![AFTER](https://user-images.githubusercontent.com/1637395/186769329-8dbe8671-38ee-42a7-84fe-22933a44ab34.gif)